apps/innbyggerpanelet/src/app/components/misc/propertieValueField/PropertieValueField.tsx

```diff
@@ -0,0 +1,27 @@
+import { Heading } from '@navikt/ds-react'
+import React, { ReactElement, useEffect, useState } from 'react'
+
+import style from './PropertieValueField.module.scss'
+
+export default function PropertieValueField(
+    {
+        propertie,
+        value
+    } : {
+        propertie: string
+        value?: string
+    }
+): ReactElement {
+    const [propertieValue, setPropertieValue] = useState<string | undefined>(value)
+
+    if (propertieValue === undefined || propertieValue === '') {
+        setPropertieValue('-')
+    }
```

Comment:
insightTechnique from Insight is set to be non-nullable, so this should not be necessary with the current props. Is this component intended to be used with several different types?

Reply:
It is intended to be used and is currently being used with several different types, not only insightTechnique :)

Reply:
I fixed it!